cocos/tween/tween-action.ts

```diff
@@ -161,9 +162,29 @@ export class TweenAction<T> extends ActionInterval {
         this.initWithDuration(duration);
     }
 
+    get relative (): boolean {
+        return !!this._opts!.relative;
+    }
+
     clone (): TweenAction<T> {
+        const action = new TweenAction(this._duration, this._originProps, this._opts);
+        action._reversed = this._reversed;
+        action.workerTarget = this.workerTarget;
+        action._id = this._id;
+        this._cloneDecoration(action);
+        return action;
+    }
+
+    reverse (): TweenAction<T> {
+        if (!this._opts!.relative) {
+            warn('reverse: could not reverse a non-relative action');
+            return this.clone();
```

Comment:
Is it ok to add a dummy tween action.

Reply:
Good point. I think we could new an empty TweenAction here.

Reply:
Updated